framework/source/class/qx/ui/table/pane/Pane.js

```diff
@@ -379,6 +383,8 @@ qx.Class.define("qx.ui.table.pane.Pane",
         this._scrollContent(scrollOffset);
       } else if (onlySelectionOrFocusChanged && !this.getTable().getAlwaysUpdateCells()) {
         this._updateRowStyles(onlyRow);
+      } else if (onlyRow) {
```

Comment:
I think this is zero-relative, so a value of 0 means do it, whereas a value of undefined or false means don't do it. Maybe this test should if `else if (typeof onlyRow == "number" && onlyRow >= 0)`?

Reply:
Yes, good catch, thanks